tests/test-extensions/sample-data-extension/src/main/java/org/mule/test/data/sample/extension/provider/GroupTestSampleDataProvider.java

```diff
@@ -8,7 +8,7 @@
 
 import static org.mule.test.data.sample.extension.SampleDataExtension.adaptLegacy;
 
-import org.mule.runtime.extension.api.annotation.param.Optional;
+import org.mule.sdk.api.annotation.param.Optional;
```

Comment:
make sure to have tests that cover both cases... old and new `@Optional` annotations

Reply:
Yes, I had check it. There are both cases for VP's and SampleData